source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,67 @@ Parameters
+
+   If an object on Source changes, ``my mirror --overwrite`` synchronizes and overwrites any matching file on Destination.
+
+   Without ``--overwrite``, if an object already exists on the Destination, the mirror process fails for the object that exists in both locations.
+
 .. mc-cmd:: --preserve, a
    
 
-   Preserve file system attributes and bucket policy rules of the
-   :mc-cmd:`~mc mirror SOURCE` on the
-   :mc-cmd:`~mc mirror TARGET`.
+   Preserve file system attributes and bucket policy rules of the :mc-cmd:`~mc mirror SOURCE` on the :mc-cmd:`~mc mirror TARGET`.
 
 .. mc-cmd:: --region
    
 
-   Specify the ``string`` region when creating new bucket(s) on the
-   target. 
+   Specify the ``string`` region when creating new bucket(s) on the target. 
 
    Defaults to ``"us-east-1"``.
 
 .. mc-cmd:: --remove
    
 
-   Removes extraneous object(s) on the target. 
+   Removes object(s) on the Target that do not exist on the Source. 
+
+   For example, objects A, B, and C exist on Source.
+   Objects C, D, and E exist on Target.
+
+   When running `mc mirror --remove`, objects A and B copy to Target and objects D and E are removed from the Target.
+   C already exists on both, so does not change on either. 
+
+   After the action, only objects A, B, and C exist on both the Source and the Target.
+
+   Use the `--remove` flag to keep the same set of objects on both Source and Target.
+
+   Use :mc-cmd:`~mc mirror --overwrite` or :mc-cmd:`~mc mirror --watch` if object C differs between Source and Target and the object on Source should be on both.
 
 .. mc-cmd:: storage-class, sc
    
 
-   Set the storage class for the new object(s) on the 
-   :mc-cmd:`~mc mirror TARGET`. 
+   Set the storage class for the new object(s) on the :mc-cmd:`~mc mirror TARGET`. 
          
-   See the Amazon documentation on
-   :aws-docs:`Storage Classes <AmazonS3/latest/dev/storage-class-intro.html>` 
-   for more information on S3 storage classses.
+   See the Amazon documentation on :aws-docs:`Storage Classes <AmazonS3/latest/dev/storage-class-intro.html>` for more information on S3 storage classses.
 
 .. mc-cmd:: --watch, w
    
 
-   Continuously monitor the :mc-cmd:`~mc mirror SOURCE` path and
-   synchronize changes.
+   Use ``--watch`` flag to mirror objects from Source to Target, where the Target may also have additional objects not present on the Source.
 
    Defaults to ``0`` (all objects).
 
+   For example, object A and B exist on the watched Source.
+   Objects A, B, and C exist on the watched Target.
+   
+   A client writes object D to Source and removes object B.
+
+   After the operation, objects A and D exist on the Source.
+   Objects A, C, and D exist on the Target.
+
+   :mc-cmd:`~mc mirror --watch` implies the :mc-cmd:`~mc mirror --overwrite` flag.
+   While watching the Source, files changed on Source also update on the Target.
```

Comment:
This feels out of place here - should it be up in 221? We could break it up into three points:

- --watch continuously synchronizes files from source to destination
- The destination may have files that do not exist on the source
- --watch overwrites objects on the Destination if they match on the Source